src/godot_fmod.cpp

```diff
@@ -1313,6 +1316,24 @@ Node * Fmod::getObjectAttachedToInstance(uint64_t instanceId) {
     return node;
 }
 
+void Fmod::setSystemDSPBufferSize(unsigned int bufferlength, int numbuffers) {
+    if (bufferlength > 0 && numbuffers > 0 && ERROR_CHECK(coreSystem->setDSPBufferSize(bufferlength, numbuffers))) {
+        GODOT_LOG(0, "FMOD Sound System: Successfully set DSP buffer size")
+    } else {
+        GODOT_LOG(2, "FMOD Sound System: Failed to set DSP buffer size :|")
+    }
+}
+
+int Fmod::getSystemDSPBufferLength() {

```

Comment:
I'm not sure if we should not return an array containing lenght and num instead of having two getter that does same things but only return partial information ?

Reply:
I had a long think about this: from a usability perspective, it will be quite confusing to Godot users to suddenly get an `Array` as return type as it is not very explicit what these values actually mean. On the contrary, it is not performant to request both values separately, however, the FMOD C++ binding forces us to pass in both by reference always.

If you are happy I can make that change to return an `Array`, however, it may get confusing to users what these values mean.

Reply:
I'm more on the side of having 2 getters. It's not ike we need performances for that, it's just a simple API call. So it's better to favor usabiity.

Reply:
Pushed!